connectors/totalenergies/src/index.js

```diff
@@ -258,15 +258,15 @@ class TemplateContentScript extends ContentScript {
     this.log('getContract starts')
     const contractElement = document.querySelector('.cadre2')
     const offerName = contractElement.querySelector('h2').innerHTML
-    const rawStartDate = contractElement.querySelector('p[class="txt-gras"]').innerHTML
+    const rawStartDate = contractElement.querySelector('p[class="font-700"]').innerHTML
```

Comment:
That looks quite fragile, but I guess this is inevitable with client-side konnectors :man_shrugging:

Reply:
The only thing I may try to avoid this is to get the parent and go down in it's child, but i'm pretty sure this is not robust enough too ...